DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py

```diff
@@ -96,44 +100,48 @@
 trackCandInputTag['muonSeededStepInOut'] = cms.InputTag("muonSeededTrackCandidatesInOut")
-trackSeedSizeBin ['muonSeededStepInOut'] = cms.int32(15)
+trackSeedSizeBin ['muonSeededStepInOut'] = cms.int32(30)
 trackSeedSizeMin ['muonSeededStepInOut'] = cms.double(-0.5)                         
-trackSeedSizeMax ['muonSeededStepInOut'] = cms.double(14.5)
+trackSeedSizeMax ['muonSeededStepInOut'] = cms.double(29.5)
+TCSizeMax        ['muonSeededStepInOut'] = cms.double(199.5)
 clusterLabel     ['muonSeededStepInOut'] = cms.vstring('Strip')
-clusterBin       ['muonSeededStepInOut'] = cms.int32(500)
+clusterBin       ['muonSeededStepInOut'] = cms.int32(100)
 clusterMax       ['muonSeededStepInOut'] = cms.double(100000)
 
 seedInputTag     ['muonSeededStepOutIn'] = cms.InputTag("muonSeededSeedsOutIn")
 trackCandInputTag['muonSeededStepOutIn'] = cms.InputTag("muonSeededTrackCandidatesOutIn")
-trackSeedSizeBin ['muonSeededStepOutIn'] = cms.int32(15)
+trackSeedSizeBin ['muonSeededStepOutIn'] = cms.int32(30)
 trackSeedSizeMin ['muonSeededStepOutIn'] = cms.double(-0.5)                         
-trackSeedSizeMax ['muonSeededStepOutIn'] = cms.double(14.5)
+trackSeedSizeMax ['muonSeededStepOutIn'] = cms.double(29.5)
+TCSizeMax        ['muonSeededStepOutIn'] = cms.double(199.5)
 clusterLabel     ['muonSeededStepOutIn'] = cms.vstring('Strip')
-clusterBin       ['muonSeededStepOutIn'] = cms.int32(500)
+clusterBin       ['muonSeededStepOutIn'] = cms.int32(100)
 clusterMax       ['muonSeededStepOutIn'] = cms.double(100000)
 
 seedInputTag     ['muonSeededStepOutInDisplaced'] = cms.InputTag("muonSeededSeedsOutInDisplaced")
 trackCandInputTag['muonSeededStepOutInDisplaced'] = cms.InputTag("muonSeededTrackCandidatesOutInDisplacedg")
-trackSeedSizeBin ['muonSeededStepOutInDisplaced'] = cms.int32(15)
+trackSeedSizeBin ['muonSeededStepOutInDisplaced'] = cms.int32(30)
 trackSeedSizeMin ['muonSeededStepOutInDisplaced'] = cms.double(-0.5)                         
-trackSeedSizeMax ['muonSeededStepOutInDisplaced'] = cms.double(14.5)
+trackSeedSizeMax ['muonSeededStepOutInDisplaced'] = cms.double(29.5)
+TCSizeMax        ['muonSeededStepOutInDisplaced'] = cms.double(199.5)
 clusterLabel     ['muonSeededStepOutInDisplaced'] = cms.vstring('Strip')
-clusterBin       ['muonSeededStepOutInDisplaced'] = cms.int32(500)
+clusterBin       ['muonSeededStepOutInDisplaced'] = cms.int32(100)
 clusterMax       ['muonSeededStepOutInDisplaced'] = cms.double(100000)
 
 seedInputTag     ['jetCoreRegionalStep'] = cms.InputTag("jetCoreRegionalStepSeeds")
 trackCandInputTag['jetCoreRegionalStep'] = cms.InputTag("jetCoreRegionalStepTrackCandidates")
-trackSeedSizeBin ['jetCoreRegionalStep'] = cms.int32(15)
+trackSeedSizeBin ['jetCoreRegionalStep'] = cms.int32(100)
 trackSeedSizeMin ['jetCoreRegionalStep'] = cms.double(-0.5)                         
-trackSeedSizeMax ['jetCoreRegionalStep'] = cms.double(14.5)
+trackSeedSizeMax ['jetCoreRegionalStep'] = cms.double(199.5)
```

Comment:
@slava77

Reply:
+1
but is this enough ?
especially for a typical JetHT PD 2017 run